gcsio/src/main/java/com/google/cloud/hadoop/gcsio/StorageProvider.java

```diff
@@ -67,14 +67,15 @@ synchronized Storage getStorage(
       storage = createStorage(credentials, storageOptions, null, null, downscopedAccessTokenFn);
       cache.put(key, storage);
       storageToCacheKeyMap.put(storage, key);
-      logger.atFinest().log(
-          "Cache miss, created new storage client. Cache hit count : %d, Cache hit rate : %.2f",
-          cache.stats().hitCount(), cache.stats().hitRate());
+      logger.atInfo().log(
+          "Cache miss for %d, created new storage client. Cache hit count : %d, Cache hit rate : %.2f",
+          key.hashCode(), cache.stats().hitCount(), cache.stats().hitRate());
     } else {
-      logger.atFinest().log(
-          "Cache hit, reusing the storage client. Cache hit count : %d, Cache hit rate : %.2f",
-          cache.stats().hitCount(), cache.stats().hitRate());
+      logger.atInfo().log(
```

Comment:
since we expect this to happen a lot, let's make this fine

Reply:
Done.